project-base/config/packages/cache.yaml

```diff
@@ -0,0 +1,15 @@
+framework:
+    cache:
+        pools:
+            main_friendly_url_slug_cache:
+                adapter: cache.adapter.redis
+                provider: snc_redis.main_friendly_url_slugs
+            shopsys.shop.product.bestselling_product.cache_provider:
```

Comment:
I am thinking about the naming, the "shop" keyword seems like a relic from the ancient times when "ShopBundle" ruled the world :slightly_smiling_face: Few lines above, there is simply `main_friendly_url_slug_cache` so is not it possible to rename it to something like this? E.g. `bestselling_product_cache`? :thinking:

Reply:
changed in 8378fe660, thanks!